sql/migrations/0001_initial.sql

```diff
@@ -17,10 +17,23 @@ CREATE INDEX ballots_by_layer ON ballots(layer);
 
 CREATE TABLE layers (
     id INT PRIMARY KEY,
-    hare_output VARCHAR
+    hare_output VARCHAR,
+    hash CHAR(32),
+    aggregated_hash CHAR(32)
 ) WITHOUT ROWID;
 
 CREATE TABLE mesh_status (
     status SMALL INT PRIMARY KEY,
     layer INT NOT NULL
-) WITHOUT ROWID;
\ No newline at end of file
+) WITHOUT ROWID;
+
+CREATE TABLE rewards (
```

Comment:
why not create 0002 file? wouldn't sqlite check that you already have version 1?

Reply:
it should not be 0002. everything before mainnet launch goes into 0001